contracts/margin/external/ERC20/ERC20PositionWithdrawer.sol

```diff
@@ -120,4 +120,32 @@ contract ERC20PositionWithdrawer is ReentrancyGuard
 
         return (tokensWithdrawn, tokensReturned);
     }
+
+    /**
+     * Withdraw WETH tokens from a position, unwrap the tokens, and send back to the trader
+     *
+     * @param  erc20Position  The address of the ERC20Position contract to withdraw from
+     * @return                The amount of ETH withdrawn
+     */
+    function withdrawAsEth(
+        address erc20Position
+    )
+        external
+        returns (uint256)
+    {
+        // verify that WETH will be withdrawn
+        address token = ERC20Position(erc20Position).heldToken();
+        require(
+            token == WETH,
+            "ERC20PositionWithdrawer#withdrawAsEth: Withdrawn token must be WETH"
+        );
+
+        // withdraw tokens
+        uint256 amount = ERC20Position(erc20Position).withdraw(msg.sender);
+        assert(WETH9(token).balanceOf(address(this)) >= amount);
```

Comment:
Not sure if this is necessary. It'll cost a reasonable amount of gas (external call + storage read), and the `WETH9(token).withdraw(amount);` will just fail if it doesn't have the balance anyways

Reply:
```suggestion
```

Reply:
Trying out this Suggested Change (beta) thing lol. Removing